src/components/Thumbnail.tsx

```diff
@@ -17,18 +18,53 @@ const StyledImage = styled.img`
 interface thumbnailProps {
   resourceIdentifier: string;
   alt: string;
+  tempContentIdentifier?: string;
```

Comment:
trengs egentlig tempContentIdentifier i denne komponenten ? Kan ikke props bare ta inn en generell identifier ? Det ser ut som om det eneste den brukes til er å bygge opp url'en - og da er logikken lik for begge ids

Reply:
Fjernet, hadde i utgangspunktet med den for å tydeliggjøre at man kan bruke samme komponent for å vise fram content thumbnails ikke bare default thumbnail.